src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.scss

```diff
@@ -141,7 +141,7 @@ $bp-largest: 1440px;
 }
 
 .news-text {
-  margin-left: 74px;
+  margin-left: 3rem;
   color: #000;
   text-align: left;
```

Comment:
could you put this color to some var or use one of available?

Reply:
I didn't changed any of colors. Only margin value has been changed

Reply:
but I can change this color as well :)

Reply:
i know) but in debugging process if we see things like this, will be better to change it

Reply:
Since this line of code is not impacting anything at all - I've removed that with this commit dee149d5def3ee2df22f604d0e37cb7a4e17dd8a